README.md

```diff
@@ -1,8 +1,15 @@
+# PayMeLah
+
 [![CI Status](https://github.com/AY2223S1-CS2103T-W13-3/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2223S1-CS2103T-W13-3/tp/actions)
+[![codecov](https://codecov.io/gh/AY2223S1-CS2103T-W13-3/tp/branch/master/graph/badge.svg)](https://codecov.io/gh/AY2223S1-CS2103T-W13-3/tp)
```

Comment:
Is this causing the CI to fail?
Not sure if the codecov badge needs to be here tbh